docs/data-access/data-download.md

```diff
@@ -0,0 +1,89 @@
+---
+id: data-download
+title: How to Download Data
+---
+
+The ARGO Data Platform uses the score-cient as a file download manager. Score facilitates the transfer of data with resumable download and built in BAM/CRAM slicing to make data download transfer fast and smooth.
```

Comment:
do we want to link "score-client" to https://www.overture.bio/products/score

Reply:
I think it would be confusing for general download users to be pointed to the overture score page.  

Its not in scope for them to want to download and install the score/song infrastrucyre.

Reply:
ok sounds good